raiden-cli/src/raiden.ts

```diff
@@ -108,10 +110,26 @@ function parseArguments() {
         default: 'auto',
         desc: 'Force a given PFS to be used; "auto" selects cheapest registered on-chain',
       },
+      pathfindingMaxPaths: {
+        type: 'number',
+        default: 3,
+        desc: 'Set maximum number of paths to be requested from the path finding service.',
+      },
+      pathfindingMaxFee: {
+        type: 'string',
+        // cast to fool TypeScript on type of argv.pathfindingMaxFee, decoded by Raiden.create
+        default: ('50000000000000000' as unknown) as UInt<32>,
+        desc: 'Set max fee per request paid to the path finding service.',
+      },
+      pathfindingIouTimeout: {
+        type: 'number',
+        default: 200000,
+        desc: 'Number of blocks before a new IOU to the path finding service expires.',
+      },
```

Comment:
Why do we need to redefine the defaults here and don't rely on the SDKs RaidenConfig defaults?

Reply:
To have it shown in `--help`. If you leave the default as `undefined`, SDK will properly fill them up, but then it doesn't get shown on the help text.

Reply:
Hmm. Could you then import the default config object from the SDK and use the values here please?

Reply:
Not feasible here, the default values aren't exported, but not critical either, it's "safe" to have different default values in the SDK and from an app using the SDK.